src/Exception/InvalidConfigException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types=1);
+
+namespace Yiisoft\Widget\Exception;
+
+class InvalidConfigException extends \Exception
+{
+    /**
+     * @return string the user-friendly name of this exception
```

Comment:
Maybe you wanted to implement `FriendlyException` here?

Reply:
yes but it would add a dependency more than you think ?

Reply:
We should make all exceptions FriendlyException ?

Reply:
No, not necessary all of them.